docs/install.md

```diff
@@ -49,9 +74,9 @@ pip install 'panpipes[spatial]'
 ```
 The extra `[spatial]` includes squidpy, cell2location, and tangram-sc packages.
 
-#### 2. Nightly versions of panpipes
+##### Nightly version of panpipes
```

Comment:
I would suggest deleting this whole subsection. Then we have two options for installing panpipes using a conda environment:
1. Clone repository → create conda env using preconfigured file → install dependencies from repository via pip
2. Create conda env manually → Install R dependencies → Install panpipes via PyPi

Currently, we describe a lot of different options for installation, and to me, it feels like we make things unnecessarily complex.